test/test_01_02.py

```diff
@@ -0,0 +1,25 @@
+from typing import Sequence
+
+import pytest
+
+from hm_1.tasks.task_01_02 import check_fibonacci
+
+
+@pytest.mark.parametrize(
+    "value",
+    [[0, 1], [0, 1, 1], [0, 1, 1, 2], [0, 1, 1, 2, 3, 5, 8], [0, 1, 1, 2, 3, 5, 8, 13]],
+)
+def test_sequence_is_fibonacci_sequence(value: Sequence[int]):
+    assert check_fibonacci(value) is True
+
+
+@pytest.mark.parametrize(
+    "value",
+    [[0], [4], [1], [1, 1], [1, 1, 2], [0, 1, 1, 3], [0, 1, 2, 1, 2, 3, 5, 8]],
```

Comment:
Why [0, 1] is fib and [0] is not?

Reply:
Because [0,1]-it's a sequence, and [0] is not, although it's a fibonacci number, in function we check that given sequence is a sequence of numbers fibonacci, [0]-it's just number, not the sequence, because of it I think we shouldn't consider this case as true fibonacci sequence.

Reply:
The sequence of size 1 is perfectly fine.  `1` is int,  `[1]` is the list of ints.  So in that case we should treat it as a sequence.